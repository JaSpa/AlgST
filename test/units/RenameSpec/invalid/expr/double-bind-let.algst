let (a, a) = global in a
