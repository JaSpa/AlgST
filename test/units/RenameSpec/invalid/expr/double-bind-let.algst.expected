
1:6: error:
	 Conflicting definitions for a
	 Bound at: 1:6
	           1:9